open_spiel/algorithms/infostate_tree.cc

```diff
@@ -82,13 +82,19 @@ const std::vector<Action>& InfostateNode::legal_actions() const {
   return legal_actions_;
 }
 
+size_t InfostateNode::corresponding_states_size() const {
+  return corresponding_states_.size();
```

Comment:
Because you seem to care about performance, consider moving one-liners into the header so that they could be inlined.